src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3331,7 +3331,7 @@ public override SyntaxNode BaseExpression()
         public override SyntaxNode TypedConstantExpression(TypedConstant value)
             => ExpressionGenerator.GenerateExpression(value);
 
-        protected override SyntaxNode GenerateExpression(ITypeSymbol? type, object? value, bool canUseFieldReference)
+        private protected override SyntaxNode GenerateExpression(ITypeSymbol? type, object? value, bool canUseFieldReference)
```

Comment:
I don't believe this was intended to be a public API, but should be confirmed with Cyrus as part of #65746

Reply:
SyntaxGenerator is not publicly inheritable (it has tons of internal abstract members).  As such, all 'protected' members are save to change in any fashion.

Note: i would love it public-api-analyzer understood this.  I think it understands if there is an explicit internal constructor, but not if there are internal abstracts.